tests/framework/unit_tests/CustomDrivers/TestPythonRaven.py

```diff
@@ -61,11 +61,11 @@ def test_findFileCWD(self):
     target = self.raven._findFile(fname)
     self.assertEqual(targetWorkflow, target)
 
-  def test_findFileFramework(self):
-    """ Find file given path relative to RAVEN framework """
-    fname = os.path.join('..', 'tests', 'framework', 'basic.xml')
-    target = self.raven._findFile(fname)
-    self.assertEqual(targetWorkflow, target)
```

Comment:
@PaulTalbot-INL Hm, I am not sure what this should be doing with a pip install?  Basically, this is finding something relative to the ravenframework which will be in site-packages, which definitely will not have a tests directory.

Reply:
Hm, yeah. I think we added this to simplify finding tests relative to a directory we knew would exist, which is basically only the RAVEN directory. I don't know what the future of the RAVEN tests are, but the answer to this question is probably dependent on that answer.

Reply:
A user who just runs pip install raven is not going to run the tests right? For developers who clone the repository, if they pip install editable, this will still work. It would probably be better if there were a utility function that locates the correct path.

Reply:
Okay, I will leave this test in, it will be one of the ones that just fails when running tests with a pip install.

Reply:
@PaulTalbot-INL or @dgarrett622  This pull request is ready to review if either of you wants to.  Thanks.